cetmix_tower_webhook/readme/DESCRIPTION.md

```
@@ -0,0 +1,5 @@
+This module implements incoming webhooks for [Cetmix Tower](https://cetmix.com/tower). Webhooks are authorised using customisable authenticators which can be pre-configured and reused across multiple webhooks. Webhooks and authenticators can be exported and imported using YAML format, which makes them easily sharable.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)